world-manager.js

```diff
@@ -1,25 +1,68 @@
 const WorldManager = (function () {
   return {
     levelHeight: 24,
-    levelWidth: 32,
+    levelWidth: 40,
     level1Update() {
       let walls = game.add.group();
       walls.enableBody = true;
+      let wall;
 
-      let wall = walls.create(64, 64, 'wall');
-      wall.body.immovable = true;
+      //vertical border walls
+      for(let w = 0; w < 24; w++){
```

Comment:
Change `24` to `WorldManager.levelHeight` (or if that doesn't work try `this.levelHeight`);